### PR TITLE
chap 6 model

### PR DESCRIPTION
**Pull request by camnguyenitepu**

### CODE REVIEW

**Comment by camnguyenitepu** [comment]
ready

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  validates :name, presence: true, length: {maximum: 50}
+  validates_format_of :email,
+                      :with => /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i,
```
Comment (thangtx88) [inline app/models/user.rb]: ident

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+                      :on => :update
```
Comment (thangtx88) [inline app/models/user.rb]: ident

**Comment by thangtx88** [comment]
commented

**Pull request closed without merging**